test/unit/packwerk/cli_test.rb

```diff
@@ -296,5 +296,17 @@ def show_strict_mode_violations(_offenses)
 
       cli.execute_command(["check", "--offenses-formatter=default", "--packages=components/platform"])
     end
+
+    test "#parse_run (private) parses parallel option and overrides the configuration" do
```

Comment:
Can we avoid calling the private method by calling the public interface and asserting the configuration object that we are already passing changed its value?

Reply:
Thank you for writing tests against the public interface! That means I can refactor without needing to change tests! 🙇🏻‍♂️